.eslintrc

```diff
@@ -13,6 +13,11 @@
   "parser": "babel-eslint",
   "rules": {
     "arrow-parens": ["error", "always"],
+    // Beware about turning this rule back on. The rule encourages you to create
+    // static class methods on React components when they don't use `this`.
+    // However, it will not warn you if you access `this` in a static method
+    // accidentally (which would most likely be a mistake).
+    "class-methods-use-this": "off",
```

Comment:
I was curious so I looked this up. Static methods in JS have access to `this`. E.g you can call another static method from a static method, or access other static properties.

But, static methods are only enumerable as properties of the class object. You can't call a static method as a property of an instance (unlike Python which allows both). Because of that the `this` in a static method is a reference to the object and not ever the instance.

See http://jsbin.com/dubelog/edit?js,console for an example.

It makes sense we don't want to start converting methods that don't use `this` into static methods because that will probably cause breakage.